manifests/init.pp

```diff
@@ -6,6 +6,7 @@
 #
 class drbd(
   $service_enable = true,
+  $service_ensure = running,
```

Comment:
can you please add a datatype to the param? We require them for new parameters.

Reply:
datatypes added